blocks/tags/tags.js

```diff
@@ -0,0 +1,12 @@
+/**
+* decorates the tags block
+* @param {Element} block The social block element
+*/
+export default async function decorate(block) {
+  const tags = block.querySelectorAll('a');
+  block.innerHTML = '';
+
+  [...tags].forEach((tag) => {
```

Comment:
We already have the tags in metadata block. Could we use them? One difference though is that you need links for them. But it looks like you can build it from the tag name itself. If not, we could keep it in a spreadsheet.

Reply:
Yes, i can check that. Good point